src/gamma.rs

```diff
@@ -280,4 +316,29 @@ mod tests {
         let z = x.iter().map(|&x| x.inc_gamma(p)).collect::<Vec<_>>();
         assert::close(&z, &y, 1e-12);
     }
+
+    #[test]
+    fn trigamma() {
```

Comment:
It would be great if you could arrange it how it’s in the other tests: one array contains inputs, and another array contains expected outputs.

Reply:
done